nanobox/backup-scripts/postgres9.sh

```diff
@@ -0,0 +1,58 @@
+# This script should be used to backup your postgresql 9 database
```

Comment:
```bash
#!/usr/bin/env bash

set -euo pipefail

# This script should be used to backup your postgresql 9 database
```
Saknar en shebang högst upp i filen

Reply:
Saknar `set -e` för att stoppa skriptet vid fel.
När man ändå håller på kan man lägga in.  
`set -u` för att kasta fel för otilldelade variabler,  
`set -o pipefail` för att inte bara stoppa vid fel i sista kommandot i en pipe, utan för alla delar.  
Och vill man att varje kommando som körs ska skrivas ut så finns `set -x`.

Jag föreslår: `set -euo pipefail` eller `set -euxo pipefail`

🎓 https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/